docs/recipes/delete-files-folder.md

```diff
@@ -70,13 +70,20 @@ var del = require('del');
 var vinylPaths = require('vinyl-paths');
 
 gulp.task('clean:tmp', function () {
+  var vp = vinylPaths();
   return gulp.src('tmp/*')
+    .pipe(vp)
```

Comment:
No, then you have to glob them again. Please see my first comment for a good solution.